docs/REST API/rest-api.md

```diff
@@ -3630,6 +3642,80 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Lock host
+
+_Available in Fleet Premium_
+
+Sends a command to lock the specified macOS, Linux, or Windows host. The host is locked once it comes online.
+
+`POST /api/v1/fleet/hosts/:id/lock`
```

Comment:
@noahtalerman @rachaelshaw just to confirm, this endpoint supercedes/deprecates the old `POST /api/v1/fleet/mdm/hosts/:id/lock`, right? It will remain supported but only for macOS?